scalafmt-core/shared/src/main/scala/org/scalafmt/util/GitOps.scala

```diff
@@ -45,36 +45,41 @@ class GitOpsImpl(workingDirectory: AbsoluteFile) extends GitOps {
             "-r",
             "HEAD",
             "--name-only",
+            "--full-name",
```

Comment:
`--full-name` gives back the name of the files "from the git root". This allows us to just append it to the `rootDir`.